vm/src/any/memory/spaceSize.hh

```diff
@@ -32,14 +32,14 @@ oop get_current_space_sizes_prim(oop ignored, slotsOop proto, void *FH);
 oop get_default_space_sizes_prim(oop);
 
 // mustn't cross the 512Mb boundary; can't map in there on SS-1s and 2s
-const caddr_t HeapStart=        (caddr_t)(64*M);
```

Comment:
Windows's address space randomization is a real pain, but this address seems to be safe without sacrificing too much empty space. It would still be better to write code that searches at runtime for a big enough free area, and I'll do that at some point.